libexec/probes/worker-scripts/uploader/uploader.py

```diff
@@ -167,19 +167,20 @@ def readMetaData(self, md):
             else:
                 summaries[eventype] = []
             # Skip reading data points for certain event types to improv efficiency  
-            if eventype not in self.allowedEvents:                                                                                                  
+            if eventype not in self.allowedEvents:                                                                                            

```

Comment:
Why is there a million spaces behind the ":"?

Reply:
Good catch @djw8605, I do not know how that got there but it is fixed now